borg/helpers.py

```diff
@@ -519,6 +519,30 @@ def dir_is_tagged(path, exclude_caches, exclude_if_present):
     return tag_paths
 
 
+def format_line(formatstr, data):
+    """Filter out unwanted properties of str.format(), because "formatstr"
+    is user provided.
+    """
+    try:
+        return(formatstr.format(**data))
+    except KeyError as e:
+        print('Error in lineformat: "{}" - reason "{}"'.format(formatstr, str(e)))
+    except ValueError as e:
+        print('Error in lineformat: "{}" - reason "{}"'.format(formatstr, str(e)))
+    except:
+        print('Line format error')
```

Comment:
in the above exception handlers, you include the formatstr in the error message. so you could do the same here.